incubator/webpagetest-agent/templates/deployment.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: extensions/v1beta1
```

Comment:
Could you use the API version of `apps/v1beta2` instead. While `apps/v1` is out it's only supported in the latest release (1.9) and `extensions/v1beta1` is deprecated and will be removed from kubernetes soon.

You can read more about apps/v1beta2 at https://kubernetes.io/docs/api-reference/v1.9/#deployment-v1beta2-apps